host/vulkan/VkDecoderGlobalState.cpp

```diff
@@ -1921,6 +1921,15 @@ class VkDecoderGlobalState::Impl {
         }
 #endif
 
+        // VK_EXT_QUEUE_FAMILY_FOREIGN_EXTENSION_NAME is emulated by gfxstream
+        if (!hasDeviceExtension(properties, VK_EXT_QUEUE_FAMILY_FOREIGN_EXTENSION_NAME)) {
```

Comment:
This requires checking in `shouldPassthrough`, as otherwise it will directly return `vkEnumerateDeviceExtensionProperties` results. 

On the other hand, this looks like we should never pass the call through and always modify the output with values from `kEmulatedDeviceExtensions`.